packages/tree/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "bl-tree",
+  "name": "@blaze-elements/tree",
   "version": "1.2.0",
   "main": "dist/main.js",
```

Comment:
there is no `main.js` generated during prod build, pls change in all packages to `"main": "dist/index.js"`